composer.json

```diff
@@ -12,9 +12,12 @@
 	"autoload": {
 		"files": [
 			"load.php"
-		]
+		],
+		"classmap": [ "inc/", "inc/stream/" ]
```

Comment:
Not a biggy, should we keep this consistent with the above eg. separate lines for each item?